### PR TITLE
Lesson 5

### PR DESCRIPTION
**Pull request by haraldF**

### CODE REVIEW

File: lesson5.md [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+sayHello();
+```
+
+---
+
+### calling a function
+
+```js
+function greet(name){
+  console.log("Hello " + name + "!");
```
Comment (pektinasen) [inline lesson5.md:417]: Have we introduced string concatenation yet? (I don't know)

File: lesson5.md [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+```js
+function min(number1, number2) {
+  if (number1 <= number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+}
+
+console.log('min of 1 and 2 is ' + min(1, 2));
```
Comment (pektinasen) [inline lesson5.md:498]: Before you used " for your strings

File: lesson5.md (hunk 1 above)
Comment (haraldF) [reply lesson5.md:417]: https://redi-school.github.io/js-berlin-2020-fall/#/29 :)

File: lesson5.md (hunk 2 above)
Comment (haraldF) [reply lesson5.md:498]: Just to remind people that both " and ' are valid :) https://redi-school.github.io/js-berlin-2020-fall/#/23

Comment (pektinasen) [reply lesson5.md:498]: ok.

Other point: are we explaining why we can `+` a string with a number? I think we haven't done that before. Do we say something about type coercion in general at this point?

File: lesson5.md [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+
+---
+
+
+(regular day: 1€, extra: 3€)
+
+The employee gets, depending on the bonus:
+* a pay raise, if more than 250€ bonus.
+* nothing, if less than 100€.
+* a chocolate box, for any other bonus.
```
Comment (sevtapim) [inline lesson5.md]:
```suggestion
The employee gets:
* a pay raise, if the total bonus is more than 250€.
* nothing, if the total bonus is less than 100€.
* a chocolate box, for any other bonus.
```

File: lesson5.md [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+to execute a function, we write the function name, and then `()`, we pass any parameters we want into the `()`
+
+```js
+function sayHello(){
+  console.log("Hello!");
+}
+
+// the function sayHello
+// does not require any parameters
+// that is why the parenthesis () are empty.
```
Comment (sevtapim) [inline lesson5.md]:
```suggestion
to execute a function, we write the function name, and then `()`, we pass any values we want into the `()`

```js
function sayHello(){
  console.log("Hello!");
}

// the function sayHello
// does not require any values, it has no parameters
// that is why the parenthesis () are empty.
```

File: lesson5.md [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+
+console.log("28 degrees: " + fahrenheitToCelsius(28));
+console.log("32 degrees: " + fahrenheitToCelsius(32));
+```
+
+---
+
+### Exercise
+
+Write a `min` function that returns the smaller of the two parameters:
```
Comment (sevtapim) [inline lesson5.md]:
```suggestion
Write a `min` function that returns the smaller of the two numbers:
```

File: lesson5.md (hunk 2 above)
Comment (haraldF) [reply lesson5.md:498]: We told the students not to do it ;) OK - I'll mention it

**Pull request merged**